packages/lab/src/number-input/NumberInput.css

```diff
@@ -279,9 +279,16 @@
 }
 
 /* Styles applied to number buttons */
+.saltNumberInput-increment {
```

Comment:
FYI... we previously had these as separate and combined..
Leave as-is for this PR.